docs/self-managed/platform-deployment/platform-8-deployment.md

```diff
@@ -0,0 +1,64 @@
+---
+id: platform-8-deployment
+title: "Camunda Platform 8 deployment"
+sidebar_label: "Overview"
```

Comment:
Do you think the sidebar label still makes sense as "Overview"? My gut says yes since you'll have the proper context around the rest of the sidebar items.

Reply:
I actually think clarifying this would be better as it's more of an "action" doc on deployment -- how do you feel about naming the sidebar label "Deployment"?

Reply:
Oh I like that!